timelock-agent/src/main/java/com/palantir/timelock/paxos/TimeLockAgent.java

```diff
@@ -200,7 +195,7 @@ public long getLatestSchemaVersion() {
 
     // No runtime configuration at the moment.
     private void registerPaxosResource() {
-        paxosResources.resourcesForRegistration().forEach(registrar::accept);
+        paxosResources.resourcesForRegistration().forEach(registrar);
```

Comment:
non-actionable: This is correct. I prefer how the old one reads, but don't feel strongly.

Reply:
not fussed, can revert